src/exported_name.rs

```diff
@@ -28,42 +28,49 @@ use crate::attributes::Attribute;
 /// If this function is called with an item that doesn't support external names,
 /// the result is unspecified.
 pub(crate) fn item_export_name(item: &rustdoc_types::Item) -> Option<&str> {
-    if item
+    // First check for export_name attribute, as it takes precedence
```

Comment:
Let's make this tricky claim easy to back up with an authoritative source, because we have the link handy and it'll be hard to Google later.
```suggestion
    // First check for export_name attribute, as it takes precedence:
    // https://github.com/rust-lang/rust/issues/47446
```